decisions/Rhovanion.txt

```diff
@@ -566,6 +567,90 @@ political_decisions = {
 	}
 }
 
+# Culture acceptance
+political_decisions = {
+	accept_avari_RHO = {
+		picture = "kingdom_of_dorwinion"
+		potential = {
+			tag = RHO
+			population_2 = 1
+			education_1 = 1
+			NOT = { accepted_culture = avari }
+			OR = {
+				DOR_539 = { owned_by = THIS }
+				DOR_545 = { owned_by = THIS }
+				DOR_550 = { owned_by = THIS }
+				DOR_555 = { owned_by = THIS }
+				DOR_558 = { owned_by = THIS }
+			}
+		}
+		
+		allow = {
+			DOR_539 = { owned_by = THIS }
+			DOR_545 = { owned_by = THIS }
+			DOR_550 = { owned_by = THIS }
+			DOR_555 = { owned_by = THIS }
+			DOR_558 = { owned_by = THIS }
+			population_4 = 1
+			education_2 = 1
+			magic_1 = 1
+			culture_2 = 1
+			money = 10000
+			war = no
+		}
+		
+		effect = {
+			prestige = 5
+			money = -10000
+			add_accepted_culture = avari
+			any_pop = {
+				limit = {
+					OR = {
+						is_accepted_culture = yes
+						is_primary_culture = yes
+					}
+					has_pop_religion = men
+					NOT = { has_pop_culture = dorwinrim }
+				}
+				militancy = 1
+				consciousness = 2
```

Comment:
Bump all of the militancy and consciousness changes to be larger (in both directions). I would imagine a realm of entirely Men wouldn't take too kindly to Elves' power increasing. Probably militancy +4 and consciousness +6. In fact, this would likely be a good usage for scaled_militancy and scaled_consciousness, based upon support of high and low nobility

![image](https://github.com/The-Third-Age/TTA-Victoria-2/assets/33076111/c55fdabc-8527-43dc-8ef2-4a5f02819f61)